### PR TITLE
Redesign lobby

### PR DESCRIPTION
**Pull request by pastpatryk**
New simplified matches list design

image

### CODE REVIEW

File: src/components/lobby/MatchItem.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        <div className={styles.row}>
+          <span className={styles.showDetails} onClick={() => setOpen((open) => !open)}>
+            {open
+              ? `${t("lobby:list.game.less_details")} ▲`
+              : `${t("lobby:list.game.more_details")} ▼`}
+          </span>
+
+          <img className={styles.image} src={game.image} alt="game icon" />
+          <h3 className={styles.gameName}>{game.displayName || game.name}</h3>
+          <h4 className={styles.matchName}>Super game room</h4>
```
Comment (glowacki-dev) [inline src/components/lobby/MatchItem.js:107]: Can we actually allow setting custom match names?

Comment (pastpatryk) [reply src/components/lobby/MatchItem.js:107]: Yes, but I'm adding this as a separate PR

Comment (pastpatryk) [reply src/components/lobby/MatchItem.js]: I forgot to push a commit that displays `#{matchID}` instead, until the custom match name feature is implemented

File: public/locales/en/lobby.json [hunk 2]
```diff
@@ -14,7 +14,11 @@
       "full": "Full",
       "join": "Join",
       "play": "Play",
-      "your_game": "Your game"
+      "your_game": "Your game",
```
Comment (glowacki-dev) [inline public/locales/en/lobby.json:17]: In this context isn't it a "match" rather than a "game"?

Comment (pastpatryk) [reply public/locales/en/lobby.json:17]: Yes, you're right 👍

**Review by glowacki-dev — APPROVED** [review]
Looks nice 👌

**Pull request merged**